builtin/logical/pki/path_fetch.go

```diff
@@ -382,6 +446,7 @@ func (b *backend) pathFetchRead(ctx context.Context, req *logical.Request, data
 
 reply:
 	switch {
+	// To Do: Should we add `http_raw_body` and `http_content_type` as optional in the response
```

Comment:
```suggestion
```
I believe those are special parameters that will no show up in the final response.

Reply:
yep. I think I added that a long time ago. Lemme remove it.

Reply:
oh wait. I actually think these were breaking tests, which is why I addd them in the response